package.json

```diff
@@ -113,6 +111,8 @@
     "webpack-bundle-analyzer": "2.13.1"
   },
   "dependencies": {
+    "@types/node": "^11.13.0",
```

Comment:
If mandatory, then let's leave "@types/node" to "*" or ">=6".

Otherwise projects with LTS node versions 8 and 10 would get conflicting types that will break things, especially with `Buffer`.